packages/components/src/draggable/index.js

```diff
@@ -129,19 +181,19 @@ export default function Draggable( {
 
 			if ( elementRect.height > cloneHeightTransformationBreakpoint ) {
 				// Scale down clone if original element is larger than 700px.
-				cloneWrapper.style.transform = 'scale(0.5)';
 				cloneWrapper.style.transformOrigin = 'top left';
 				// Position clone near the cursor.
-				cloneWrapper.style.top = `${ event.clientY - 100 }px`;
-				cloneWrapper.style.left = `${ event.clientX }px`;
+				x = event.clientX;
+				y = event.clientY - 100;
+				// Can also be written as the affine transform: matrix( 0.5, 0, 0, 0.5, x, y ).
+				// Note that order matters, translate should be done before scale.
+				cloneWrapper.style.transform = `translate( ${ x }px, ${ y }px ) scale(0.5)`;
```

Comment:
Do we have any examples where we drag an item that's more than 700px? I was wondering if we still needed to handle this case.

Reply:
I think just about every interaction uses a draggable chip now rather than a clone of an element, so this is just here for consumers of the package.

Reply:
I'm quite tempted to rip out this case. Here's what it looks like in **trunk** with a draggable with a 701x701px square. Scaling to 50% of the original draggable isn't going to work with larger items, so folks will likely need to come up with a similar solution to what we went with.

The positioning also looks off, cursor at top left vs center (which I can fix, but would modify current behavior).

https://user-images.githubusercontent.com/1270189/125821948-92485ce6-4f6a-4907-96d8-90ce5355680e.mp4

Reply:
I tried removing this case in https://github.com/WordPress/gutenberg/pull/33395/commits/7e1bda9c910355923f16d48e2933a2dcc5485f16

Reply:
Hmm, yeah. It feels like it'd be better if it had a max size instead of reducing by 50%, but maybe that's to stop the drag image from being blurry.

Removing it seems ok, we can monitor to see if there's any negative feedback.

I imagine this was for large blocks, so the use case is more than likely no longer present.